docs/guides/resources/user-account-resources.primary.in.rst

```diff
@@ -0,0 +1,168 @@
+It is a requirement defined by the `Science Requirements Document <https://docushare.lsst.org/docushare/dsweb/Get/LPM-17>`_ that the "Data Management System will also provide at least 10% of its total capability for user-dedicated processing and user-dedicated storage" (Section 3.5).
+
+This page quantifies the individual and shared resources that every account at the US Data Access Center (US DAC) has access to, by default, via the Rubin Science Platform at |rsp-url|.
+
+.. important::
+   The US DAC, the Rubin Science Platform, and this documentation are currently under construction.
+   The estimates quoted below include current, planned, and to-be-determined (TBD) values, all of which are subject to change as hardware, software, and user habits evolve.
+
+**Page last updated:** March 17 2025
+
+
+Computational processing
+========================
+
+Computational resources are available via the Notebook Aspect (JupyterLab).
+
+Notebook server options:
+
+- Small (1.0 CPU, 4 GB RAM)
+- Medium (2.0 CPU, 8 GB RAM)
+- Large (4.0 CPU, 16 GB RAM)
+
+Only CPUs (central processing units) are available.
+No GPU (graphical processing units) are available and there is no plan to add them.
+
+- Cores per science user: 0.1 [#f1]_
+
+
+Batch processing
+----------------
+
+This refers to parallel processing (asynchronous) job submission.
+The user batch facility is focused on supporting a large variety of smaller needs for the broader community (`DMTN-202 <https://dmtn-202.lsst.io/>`_).
+User batch processing will be available by Data Release 1 (DR1).
+Access to user batch processing will be allocated by the Resource Allocation Committee.
+
+- Number of core hours total per year: 4.53E+06 [#f2]_
+- Number of core hours per user available via RAC: TBD; to be set by DR1
+
+
+Storage
+=======
+
+During the Early Science era (the Data Previews and Data Release 1), the total amount of shared user space across all user directories (home, project, and scratch) is 24 TB.
+
+During the Operations era, the anticipated amount of individual home directory disk space (in the cloud) is 0.4 TB [#f3]_.
```

Comment:
That's not what's in the sizing model, I don't think. I will double check.  But I believe it's factor of 10 off and that's only assuming 5,000 users. Let me provide some wording for this section.

Reply:
Alright I'll leave this one unresolved until we have new wording thanks

Reply:
I would like to stick to the wording in the text I wrote here if that is okay. 
https://github.com/lsst/rsp_lsst_io/pull/44/files

Reply:
Sure -- incorporated that text into "Storage" and new subsection "Scratch".